db/schema.rb

```diff
@@ -1251,7 +1253,7 @@
        LANGUAGE plpgsql
       AS $function$
                   BEGIN
-                    new.phone_index = (regexp_replace(new.phone, '\\D','', 'g'));
+                    new.phone_index = (regexp_replace(new.phone, '\D','', 'g'));
```

Comment:
Why are we changing that?

Reply:
Damnit, we shouldn't be. There's a bug in fx where is screws this up.